ferrocene/doc/user-manual/src/rustc/proxy-targets.rst

```diff
@@ -0,0 +1,273 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Proxy Targets
+=============
+
+This chapter describes how to use a Proxy Target to validate the operation of a bare-metal target.
+
+Bare-metal targets do not have an operating system (OS), this means they do not link to ``libc`` or have
+access to ``std`` functionality like ``println!("printing")`` or ``assert!()``.
```

Comment:
`assert!` is defined in libcore and thus also available in no-std context. (`assert!` calls `core::panic!` not `std::print!`)